Fix format error

Currently a lint error:

Run black --check .
would reformat /home/runner/work/indeed-data-job-scraper/indeed-data-job-scraper/src/main.py
would reformat /home/runner/work/indeed-data-job-scraper/indeed-data-job-scraper/tests/test_main.py

Oh no! 💥 💔 💥
2 files would be reformatted.
Error: Process completed with exit code 1.

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 format:
-	black *.py
+	black src/ tests/
 
 lint:
 	pylint --disable=R,C src/

--- a/src/main.py
+++ b/src/main.py
@@ -1,7 +1,9 @@
 # Dummy python script to pass linting
 
+
 def main():
     print("Hello, world!")
+
 
 if __name__ == "__main__":
     main()

--- a/tests/test_main.py
+++ b/tests/test_main.py
@@ -1,9 +1,11 @@
 import unittest
 from src.main import main
 
+
 class TestMain(unittest.TestCase):
     def test_main(self):
         self.assertIsNone(main())
+
 
 if __name__ == "__main__":
     unittest.main()

